docs/installation.rst

```diff
@@ -560,6 +561,156 @@ As mentioned before, the script engine can be started from within ``$SCRIPT_ENGI
+If your email server (e.g. Microsoft Exchange) allows to reroute traffic for a specific subdomain to another server, you can also setup a respective rule there.
+Please, make sure the email is not modified and for instance the sender address is not changed due to some forwarding mechanism.
+
+The next subsection explains how you setup the mail relay and configure it to use your email domain.
+
+Install and configure the mail relay
+````````````````````````````````````
+The mail relay comes as a JAR file which requires Java 8 for the execution.
+Please, make sure Java 8 is installed before continuing with the setup.
+
+The mail relay can be installed on the same server as the Workflow Accelerator web application or on a different server.
+If you choose two different servers, please ensure the mail relay is able to reach the server which runs the web application.
+
+#. Create a new local directory for the mail relay, e.g. ``C:\Program Files\Mail Relay`` or ``/var/lib/mail-relay``.
+
+    * We will refer to this directory as ``$MAIL_RELAY_HOME``.
+
+#. Copy the content of the directory ``$WORKFLOW_HOME/mail-relay/`` to your newly created directory.
+
+    * You should find the file ``mail-relay.jar`` directly within your directory: ``$MAIL_RELAY_HOME/mail-relay.jar``.
+
+#. Within the ``$MAIL_RELAY_HOME`` directory create a new directory ``logs``.
+
+#. Open ``$MAIL_RELAY_HOME/logback.xml`` and edit ``value`` in the following line: ::
+
+    <property name="LOG_DIR" value="."/>
+
+#. Set ``value`` to the absolute path of ``$MAIL_RELAY_HOME/logs``.
+
+    * For example ``C:\\Program Files\\Mail Relay\\logs`` or ``/var/lib/mail-relay/logs``
+    * On Windows you have to use ``\\`` as a separator for the path.
+
+#. Open ``$MAIL_RELAY_HOME/mail-relay.properties`` and edit the values for the domain, port and Workflow Accelerator URL.
+
+    * Ensure that the port for the mail relay is not used by any other application.
+    
+The configuration file offers the following options:
+
+============================    =================================================================================
+``workflow.relay.domain``       Defines the (sub)domain which is used to receive emails for Workflow Accelerator.
+``workflow.relay.serverUrl``    Defines the URL of the Workflow Accelerator web application.
+``workflow.relay.port``         Defines the port the SMTP will use and listen to.
+============================    =================================================================================
+
+After the successful setup you can start the mail relay by opening the command line and executing the following command ::
+    
+    java -jar $MAIL_RELAY_HOME/mail-relay.jar
+
+Replace ``$MAIL_RELAY_HOME`` with the absolute path to the JAR file.
+
+You can also register a new service or setup a new init script depending on your operating system to ease the management of the mail relay.
```

Comment:
It isn't clear to me exactly what 'init script' refers to here, or where it’s first mentioned.